### ISSUE TITLE
Errors when repo bucket is not in the profile's default region

### ISSUE THREAD

**Issue by coderanger** [hypnoglow/helm-s3#51]
It would be nice to process the redirect to the correct region (or allow configuring it per repo) rather than having to swap the profile's region around.

**Comment by bryankaraffa** [comment]
+1 on this.  It would be nice to have a `--repo-region` flag or similar.

If the helm chart bucket is in a different region than what's set in your ENV or AWS CLI Profile, helm-s3 can fail to find the bucket.  To workaround this currently, we need to pass the correct AWS_REGION explicitly during a `helm install` or `helm upgrade`
```sh
AWS_REGION=us-east-1 helm upgrade my-release custom-service --repo s3://my-example-helm-charts-production
```

Instead we would like to do something like this to instruct helm-s3 to look in us-east-1 for the bucket:
```
helm upgrade my-release custom-service --repo s3://my-example-helm-charts-production --repo-region us-east-1
```

**Comment by ndegory** [comment]
that's a major issue for me, I have scripts where I have to export the region based on where my Helm repo buckets are located, and this has side effects (think other plugins requiring this variable, with a different region). Since we can programmatically get the location of the bucket, shouldn't we let the s3 plugin do that instead of explicitly setting the region before we call the Helm CLI?

**Comment by ndegory** [comment]
I was looking at a way to make it transparent for the user, unfortunately the API only allows the owner of the bucket to get its location. So the user still has to pass the information to Helm.

**Comment by hypnoglow** [comment]
Merged #117 , will be available in the next plugin release.

**Comment by hypnoglow** [comment]
The plugin version 0.10.0 is released.
